docs/technical-design/endpoints.md

```diff
@@ -0,0 +1,213 @@
+# Endpoints
+
+There are two ways that we create endpoints in our app. One uses GraphQL, the other creates AWS Lambda handlers.
+
+When we want to fetch or update information relevant to the app itself--think of submissions, or user info--we'll typically use GraphQL.
```

Comment:
```suggestion
**When we want to fetch or update information relevant to the app itself--think of submissions, or user info--we'll typically use GraphQL.**
```

Reply:
Bolded these lines for emphasis --  I feel like in a long doc it can be easy to miss the key points. I feel like these two sentences are really the key point if you come here wondering "what should I do" lol.